### ISSUE TITLE
psycopg2.errors.NotNullViolation on commit c96741a7 when rows exist

### ISSUE THREAD

**Issue by mattpulver** [conbench/conbench#586]
On commit c96741a742d253d1e6b858496f665fb81e48637a running `alembic upgrade head` on a database with existing data results in an error
```
sqlalchemy.exc.IntegrityError: (psycopg2.errors.NotNullViolation) column "is_step_change" contains null values
```
One quick fix is to set a default value:
```
diff --git a/migrations/versions/4efa955b2696_add_is_step_change_column.py b/migrations/versions/4efa955b2696_add_is_step_change_column.py
index c37c336..2a84e8f 100644
--- a/migrations/versions/4efa955b2696_add_is_step_change_column.py
+++ b/migrations/versions/4efa955b2696_add_is_step_change_column.py
@@ -17,7 +17,7 @@ depends_on = None
 
 def upgrade():
     op.add_column(
-        "benchmark_result", sa.Column("is_step_change", sa.Boolean(), nullable=False)
+        "benchmark_result", sa.Column("is_step_change", sa.Boolean(), nullable=False, server_default='f')
     )
 
 
```
however an alternative is given [here](https://stackoverflow.com/a/41026374).

**Comment by austin3dickey** [comment]
Hi Matt! Thanks for the issue, sorry I introduced the bug. This should be fixed in #585 . Let me know if it doesn't for you.

**Comment by mattpulver** [comment]
Fortunately the downgrade option made this an easy fix:
```
alembic downgrade 4d8d67396f79
git checkout .
git pull
alembic upgrade head
```
Fixed. Thank you!
